pootle/apps/pootle_statistics/migrations/0012_remove_create_subs.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-17 17:54
+from __future__ import unicode_literals
+
+import logging
+import time
+
+from django.db import migrations
+
+
+logger = logging.getLogger(__name__)
+
+
+def remove_create_subs(apps, schema_editor):
+    subs = apps.get_model("pootle_statistics.Submission").objects.all()
+    to_delete = subs.filter(type=10)
+    total = to_delete.count()
+    offset = 0
+    step = 10000
+    start = time.time()
+
+    while True:
+        units = to_delete[:step]
+        to_delete.filter(id__in=set(units.values_list("id", flat=True))).delete()
+
+        logger.debug(
+            "deleted %s/%s in %s seconds"
+            % (offset + step, total, (time.time() - start)))
+        if offset > total:
+            break
+        offset = offset + step
```

Comment:
This applies to https://github.com/translate/pootle/commit/534be268120697269523032c79ea42133efb0570#diff-81b87d01574051b669a0370a4e83016dR42 as well.